src/api-reference/budget/budget-trackingfield.md

```diff
@@ -1,122 +1,159 @@
 
-### Parameters
+### Scopes
 
-    N/A
+Name | Description
+---|---
+`budgetitem.write`|Create/update/delete access to budget data
+`budgetitem.read`|Read access to budget data
 
-### Response
+### Request
 
-[Budget Tracking Field Array](#budgetTrackingField)
+#### Headers
 
-## <a name="get"></a>Retrieve a Budget Tracking Field
+* [RFC 7235 Authorization](https://tools.ietf.org/html/rfc7235#section-4.2)
+* [RFC 7231 Content-Type](https://tools.ietf.org/html/rfc7231#section-3.1.1.5)
 
-    GET  /budget/v4/budgetTrackingField/{id}
+#### Parameters
 
-### Parameters
+N/A
 
-| Name | Type     | Format | Description                                        |
-| ---- | -------- | ------ | -------------------------------------------------- |
-| id   | `string` | `path` | The budget tracking field's key field (sync guid). |
+##### URI Template
+
+```shell
+GET /budget/v4/budgetTrackingField
+```
 
 ### Response
 
-[Budget Tracking Field](#budgetTrackingField)
+#### Status Codes
 
-### Example JSON Response
+* [200 OK](https://tools.ietf.org/html/rfc7231#section-6.3.1) Successful call, response is in body.
+* [400 Bad Request](https://tools.ietf.org/html/rfc7231#section-6.5.1) The request was determined to be invalid by the server. Possibly a validation failed on the data that was sent in the payload. The response will have a list of validation errors in the body. See below for an example 400 response.
+* [403 Forbidden](https://tools.ietf.org/html/rfc7231#section-6.5.3) The user does not have the necessary permissions to perform the request.
+* [404 Not Found](https://tools.ietf.org/html/rfc7231#section-6.5.4) The resource could not be found or does not exist
+* [500 Internal Server Error](https://tools.ietf.org/html/rfc7231#section-6.6.1) Error message in response body
+* [504 Gateway Timeout](https://tools.ietf.org/html/rfc7231#section-6.6.5) Error message in response body
 
-```json
-{
-  "syncGuid": "2c3dc4dc-130e-4524-91d7-a20dd4bc62e4",
-  "dataType": "VARCHAR",
-  "listSyncGuid": null,
-  "lastModifiedDate": "2018-03-22 00:54:09",
-  "status": "OPEN",
-  "budgetSequenceNumber": 1,
-  "budgetTrackingFieldDefinitions": [
-    {
-      "syncGuid": "9aea2524-a04e-44dd-a85b-cfdd5ae4cbeb",
-      "defaultItemKey": null,
-      "displayName": "Budget Tracking Code",
-      "ctrlType": "EDIT",
-      "defaultValue": "TextField",
-      "lastModifiedDate": "2017-03-22 00:54:09",
-      "connectedListSequenceNumber": 1,
-      "status": "OPEN",
-      "hierarchyCode": "1"
-    }
-  ]
-}
-```
+#### Headers
 
-## <a name="post"></a>Create/Update a Budget Tracking Field
+* `concur-correlationid` (Optional) is a Concur specific custom header used for technical support in the form of a [RFC 4122 A Universally Unique IDentifier (UUID) URN Namespace](https://tools.ietf.org/html/rfc4122)
```

Comment:
You've probably got more actual headers?

Reply:
Per our conversation, I've added a comprehensive list of headers at the end of all pages